README.md

```diff
@@ -38,7 +38,8 @@ __You should consider the [use of `--env-file=`](https://docs.docker.com/engine/
 * `DB_PORT`: port to use to connect to database. Optional, defaults to `3306`
 * `DB_USER`: username for the database
 * `DB_PASS`: password for the database
-* `DB_NAMES`: names of databases to dump; defaults to all databases in the database server
+* `DB_NAMES`: names of databases to dump (separated by space); defaults to all databases in the database server
+* `DB_NAMES_EXCLUDE`: names of databases (separated by space) to exclude from the dump; `information_schema`. `performance_schema`, `sys` and `mysql` are excluded by default. This only applies if `DB_DUMP_BY_SCHEMA` is set to `true`. For example, if you set `DB_NAMES_EXCLUDE=database1 db2` and `DB_DUMP_BY_SCHEMA=true` then theese two databases will not be dumped by mysqldump
```

Comment:
I think you meant to quote it, so 

```
if you set `DB_NAMES_EXCLUDE="database1 db2"` 
```

Reply:
Also one too many "e" in `theese`

Reply:
No quotes are needed. Just checked it and if I add quotes they will be added to the DB name. So e.g.:

DB_NAMES_EXCLUDE="db1 db2 db3"

will result in
"db1
db2
db3"